assets/js/components.js

```diff
@@ -47,6 +47,26 @@ jQuery(function () {
         $(td).html(`${rowData.attributes.github_enforce_admins_enabled}`)
       },
     },
+    {
+      data: 'attributes.environments',
+      createdCell: function (td, _cellData, rowData) {
+        const environments = rowData.attributes.environments
+        const prodEnvironments = environments ? environments.filter(env => env.name === 'prod') : []
+
+        let prodSlackChannel = ''
+        if (prodEnvironments.length === 0) {
+          prodSlackChannel = 'No Prod environment'
+        } else {
+          prodSlackChannel = prodEnvironments.map(env => {
+            if (env.alerts_slack_channel === null) {
+              return 'Missing slack channel data'
+            }
+            return `${env.alerts_slack_channel}`
```

Comment:
Also if these want to be links then you can use these as the URL and put the text in the anchor to `View` or something?

Reply:
Removed anchor part as this is channel and for anchor it needs channel id to work .